modules/admin_manual/pages/configuration/files/federated_cloud_sharing_configuration.adoc

```diff
@@ -20,39 +20,36 @@ However, to do so, you have to set `'sharing.federation.allowHttpFallback' => tr
 
 Follow these steps to establish a trusted connection between two servers.
 
-. Verify that both servers have SSL certificates.
-  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid.
+. Verify that both servers have SSL certificates
+  If you open the server URL in your browser and see a lock icon on the left-hand side of the address bar, the certificate is valid
 +
 .Lock icon in the address bars in Firefox, Google Chrome, and Safari.
 image:configuration/files/browser-address-bars.png[Lock icon in Firefox, Google Chrome, and Safari]
-. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>' setting is configured to the correct URL, instead of `localhost`, in **config.php**.
-. Reset the federation job in your `oc_jobs` table. 
-  This job is required to get the verification token from the other server to establish a federation connection between two servers. 
-  The resetting ensures that it will be executed when we run xref:configuration/server/background_jobs_configuration.adoc#cron[system:cron] later.
+. Verify that the `'overwrite.cli.url' => 'https://<SERVER_URL>'` setting is configured to the correct URL instead of `localhost` in **config.php**
+. Navigate to **admin settings -> Sharing -> Federation**
+. Add **server 1** to the trusted servers on **server 2**
+. Add **server 2** to the trusted servers on **server 1**
+. Run `system:cron` from the ownCloud directory of both servers (for example `/var/www/owncloud/`)
 +
-[source,bash]
+[source,bash,subs="attributes+"]
 ----
-mysql -u root -e "update oc_jobs set last_run=0 where class='OCA\\Federation\\SyncJob';" owncloud;
-mysql -u root -e "update oc_jobs set last_checked=0 where class='OCA\\Federation\\SyncJob';" owncloud;
+{occ-command-example-prefix} system:cron
 ----
-. Navigate to **admin settings -> sharing -> Federation**
-. Add **server 1** to the trusted servers on **server 2**.
-. Add **server 2** to the trusted servers on **server 1**.
-. Now run the cron job in your ownCloud directory (for example `/var/www/owncloud/`).
+. Run the `OCA\Federation\SyncJob` job with the `force` option on both servers (you can get the correspondent job ID by using the `background:queue:status` occ command)
 +
 [source,bash,subs="attributes+"]
 ----
-{occ-command-example-prefix} system:cron
+{occ-command-example-prefix} background:queue:execute --force 10
```

Comment:
```suggestion
{occ-command-example-prefix} background:queue:execute --force <jobId>
```

Can we just mention `<jobId>` something like this?
How do we usually indicate to users that they need to substitute their own value in a command?

Reply:
Also, I'd suggest to not only use `--force` here, (which does not force much, it still scares the user with a fat where he has to type 'yes') but use `--force --accept-warning` which really runs the command without asking questions.

Reply:
`--force` is rather meant for running the job even if it is already within the time interval. So the time interval will be ignored and `last_run` will be set to `0`. But yes, also using `--accept-warning` makes sense.

Reply:
Mind to adapt @phil-davis suggestable commit with whatever option is correct or commit the suggestion if it is ok? Else this stays open and nobody cares...